dynamic_models/schema.py

```diff
@@ -4,21 +4,24 @@
 
 
 class ModelSchemaEditor:
-    def __init__(self, initial_model=None, db_name=DEFAULT_DB_ALIAS):
+    def __init__(self, initial_model=None, db_name=DEFAULT_DB_ALIAS, is_managed=True):
```

Comment:
**suggestion**: if the model isn't managed then there's no reason to create a schema editor. Consider making the schema editor property on `ModelSchema` nullable so we can avoid all the `if self.is_managed` logic gates here.